src/components/Books.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import AddNewBook from './AddNewBook';
+import BOOKS from './data';
+
+export default function Books() {
+  const [books, setBook] = useState(BOOKS);
+  const handleDelete = (e) => {
+    const newBooks = books.filter((b) => b.id.toString() !== e.currentTarget.name);
+    setBook(newBooks);
+  };
+
+  return (
+    <div className="books">
+      <AddNewBook setBook={setBook} />
+      <ul className="books-ul">
+        {
+      books.map((b) => (
+        <li className="books-ul-li" key={uuidv4()}>
+          <h3>{b.title}</h3>
+          <button type="button" name={b.id} onClick={handleDelete}>del</button>
+        </li>
+      ))
+
+    }
+      </ul>
+    </div>
```

Comment:
Good jobs creating the books component 👍  , however for React good practice you should refactor  the <Books > component into other sub-component , so it may contain  the `<book/> ` component that has book data as a prop and renders the title and the author, etc
You could write for example

```bash
const Book = ({ book}) => (
  <li>
    {book.title}
    {' '}
    -
    {book.author}
    <button>
      Remove
    </button>
  </li>
);

// BookList.js
const BookList = ({ books}) => (
  <ul>
    {books.map((book) => (
      <BookItem key={book.id} book={book}  />
    ))}
  </ul>
);
```